### PR TITLE
Print line and column numbers for exception thrown

### LINKED ISSUE #1459: Incorrect error line numbers in wrangler dev
#### Environment

* operating system: Linux
* output of `rustc -V`: rustc 1.43.1 (8d69840ab 2020-05-04)
* output of `node -v`: v12.18.2
* output of `wrangler -V`: wrangler 1.10.3
* contents of `wrangler.toml`

#### Steps to reproduce

1. Write a script containing a syntax error.
2. Run `wrangler dev`.
3. Send a request to the local server.

#### What did you expect to see?

```
SyntaxError: missing ) after argument list
    at worker.js:135:54
```

#### What did you see instead?

(in red)

```
Uncaught
SyntaxError: missing ) after argument list
```

No line number is reported.

#### More details

The inspector protocol produces a message like this:

```
{
  "method": "Runtime.exceptionThrown",
  "params": {
    "timestamp": 0,
    "exceptionDetails": {
      "exceptionId": 1,
      "text": "Uncaught SyntaxError: missing ) after argument list",
      "lineNumber": 134,
      "columnNumber": 54,
      "url": "worker.js"
    }
  }
}
```

Looks like it's a matter of pulling the line number out and reporting it?

### PR DESCRIPTION
**Pull request by nataliescottdavidson**
closes #1459

### CODE REVIEW

File: Cargo.toml
Comment (koeninger) [inline Cargo.toml:18]: Can we wait to merge this until https://github.com/cloudflare/chrome-devtools-rs/pull/48 is merged?

**Review by koeninger — APPROVED** [review]
LGTM aside from preferring to wait until the chrome-devtools change is merged

File: Cargo.toml
Comment (nataliescottdavidson) [reply Cargo.toml:18]: Sure- btw, that's an abandoned-ish repo which used to be admin'd by Avery Harnish. I'm the admin now so I can just shove my changes through, it would be nice to get another set of eyes on it though.

**Pull request merged**
